index.html

```diff
@@ -9,30 +9,38 @@
     <title>Portfolio Project</title>
 </head>
 <body>
+    <header>
     <nav class="nav-bar">
+        <a href="#">
         <h1 class="title">Welcome!</h1>
+        </a>
         <a class="icon">
             <i class="fa fa-bars"></i>
         </a>
     </nav>
-    <section class="headline">
+    </header>
+    <section class="main-container">
+    <div class="headline">
         <div class="intro-txt">
-            <h2 class="wave-txt">Hey there. I'm Lucas</h2>
-            <h2 class="desc-txt">I'm a software developer</h2>
+            <h1 class="wave-txt">Hey there. I'm Lucas</h1>
+            <h1 class="desc-txt">I'm a software developer</h1>

```

Comment:
- [ ] Please know that a website can't have more than one `h1` tag (this only applies for `h1` tag not the rest of the `hx` tags), kindly fix that 😁

Reply:
Great suggestion ! Didn't know this, thanks